src/io/flutter/run/bazelTest/FlutterBazelTestLineMarkerContributor.java

```diff
@@ -23,7 +23,6 @@ public FlutterBazelTestLineMarkerContributor() {
   @Nullable
   @Override
   public Info getInfo(@NotNull PsiElement element) {
```

Comment:
I haven't looked at the parent impl, but is this method override necessary now?

Reply:
@jwren I believe it is only supposed to indicate that the current one might be given that I can open a non-bazel project after setting this option and everything still seems fine.
@jwren  any background on the intended behavior of this option in the Dart plugin?

Reply:
I don't have context with `FlutterBazelTestLineMarkerContributor`, did I add this, or someone else, @daveshuckerow?

Reply:
Dave did this. My take on this code is it would be nice if we short circuited earlier and didn't check whether the project was bazel every time in this traversal but that this code isn't too much of a performance problem as written given the caching.